From 10813be61657b28c6a9768f887509b8f1e1844d8 Mon Sep 17 00:00:00 2001
From: Pavan Nikhilesh <pbhagavatula@marvell.com>
Date: Tue, 4 Dec 2018 21:08:58 +0300
Subject: [PATCH 019/767] octeontx2-af: update TIM 10ns clk source min interval

commit 0477b21ab070a55aacded3c4a25b3a1a59ff7309 from
git@git.assembla.com:cavium/WindRiver.linux.git

The hardware requires a minimum of 256 ticks (2.56us) regardless of the
number of rings in use.
Previously, it was speculated that having less number of rings (<16)
allowed a minimum of 100 ticks (1us) to be configured.

Change-Id: Ie4752da67109bd8b7f6604244c5aafcb612a8c84
Signed-off-by: Pavan Nikhilesh <pbhagavatula@marvell.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_tim.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_tim.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_tim.c
index 616ceae62422..cbb45d09d5b8 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_tim.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_tim.c
@@ -172,7 +172,7 @@ int rvu_mbox_handler_tim_config_ring(struct rvu *rvu,
 
 	switch (req->clocksource) {
 	case TIM_CLK_SRCS_TENNS:
-		intervalmin = 100;
+		intervalmin = 256;
 		break;
 	case TIM_CLK_SRCS_GPIO:
 		intervalmin = 256;
-- 
2.31.1

